hydrolib/tools/extforce_convert/converters.py

```diff
@@ -281,15 +281,15 @@ def convert(self, forcing: ExtOldForcing, time_unit: str) -> Boundary:
             )
 
         forcings_list = []
-        tim_files = list(self.root_dir.glob(f"{poly_line.filepath.stem}*.tim"))
+        tim_files = list(self.root_dir.glob(f"{poly_line.filepath.stem}_*.tim"))
```

Comment:
This is not safe against, for example:
right_mister.pli (with right_mister_0001.tim) and right_mister_sal.pli (with right_mister_sal_0001.tim).
I think the change below is safer:
```suggestion
        tim_files = list(self.root_dir.glob(f"{poly_line.filepath.stem}_[0-9][0-9][0-9][0-9].tim"))
```